# Tensorflow bug 32492
- behavior: Segmentation Fault

- description: `tf.linalg.triangular_solve()` process finished with an exit code( interrupted by SIGSEGV )

- Affect version: tensorflow:2.0.0
